skyportal/tests/api/test_followup_requests_reprioritize.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from skyportal.tests import api
+
+
+def test_reprioritize_followup_request(
+    public_group_sedm_allocation, public_source, upload_data_token, super_admin_token
+):
+
+    datafile = f'{os.path.dirname(__file__)}/../data/GW190425_initial.xml'
+    with open(datafile, 'rb') as fid:
+        payload = fid.read()
+    data = {'xml': payload}
+
+    status, data = api('POST', 'gcn_event', data=data, token=super_admin_token)
+    assert status == 200
+    assert data['status'] == 'success'
+
+    dateobs = "2019-04-25 08:18:05"
+    params = {"include2DMap": True}
+
+    status, data = api(
+        'GET', f'gcn_event/{dateobs}', token=super_admin_token, params=params
+    )
+    assert status == 200
+    localization_id = data['data']['localizations'][0]['id']
+
+    request_data = {
+        'allocation_id': public_group_sedm_allocation.id,
+        'obj_id': public_source.id,
+        'payload': {
+            'priority': 1,
+            'start_date': '3020-09-01',
+            'end_date': '3022-09-01',
+            'observation_type': 'IFU',
+        },
+    }
+
+    status, data = api(
+        'POST', 'followup_request', data=request_data, token=upload_data_token
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+    id = data['data']['id']
+
+    new_request_data = {
+        'localizationId': localization_id,
+        'requestIds': [id],
+    }
+
+    status, data = api(
+        'PUT',
+        'followup_request/prioritization',
+        data=new_request_data,
+        token=upload_data_token,
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+
+    status, data = api('GET', f'followup_request/{id}', token=upload_data_token)
+    assert status == 200
+
+    assert data['data']["payload"]["priority"] == 5.0
```

Comment:
So the priorities that get assigned really depend on which followup IDs are given? If I specify only one ID that always gets priority 5, regardless of the other requests that aren't specified?

Reply:
Yeah my thinking here is that you have done some filtering and limited to the subset you care about (based on the filtering in that page). Then you want to order them by some kind of criteria, in this case the skymaps. The priority range can be changed from 1-5 in the option list, but yeah, if its just 1, it becomes top priority.